internal/testing/utils/path_test.go

```diff
@@ -0,0 +1,38 @@
+package path
+
+import (
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// TestRootPathFromCWD exercises RootPathFromCWD and RootPath
+func TestRootPathFromCWD(t *testing.T) {
+	r, err := RootPathFromCWD()
+	require.NoError(t, err)
+	_, err = os.Stat(filepath.Join(r, "LICENSE"))
+	require.NoError(t, err, "Must find a LICENSE file")
+
+	// Ensure there are no other license files in sub-directories
+	err = filepath.WalkDir(r, func(p string, d fs.DirEntry, err error) error {
+		if err != nil {
+			return err
+		}
+		// Skip the root of the project
+		w, _ := filepath.Split(p)
+		w = filepath.Clean(w)
+		if w == r {
+			return nil
+		}
+		if d.Type().IsRegular() && d.Name() == "LICENSE" {
+			return fmt.Errorf("Found an unexpected LICENSE file in a sub-directory: %s", p)
```

Comment:
```suggestion
			return fmt.Errorf("found an unexpected LICENSE file in a sub-directory: %s", p)
```
Huge showstopper, but golang dislikes errors starting with a capital and I have a yellow squiggly line which is hugely distressing

Reply:
Yeah, that's fine

Fixed 46a9d14

Reply:
I've rebased this down, since it was so small.